src/index.html

```diff
@@ -11,6 +11,25 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div data-qa="big" class="bar big">
+      <form action="#" method="get">
+        <input
+          class="inp bi"
+          data-qa="keypress"
+          type="text"
+          placeholder="Try  &quot;Los Angeles&quot;"
+        >
+      </form>
+    </div>
+    <div data-qa="small" class="bar small">
+      <form action="#" method="get">
+        <input
+          class="inp sm"
```

Comment:
It's not a good idea to short class names that way, it's hard to understand what it means